src/components/ValueCard/DataStateRenderer.jsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import classnames from 'classnames';
+import { Tooltip } from 'carbon-components-react';
+import { ErrorFilled24, WarningFilled24 } from '@carbon/icons-react';
+
+import { settings } from '../../constants/Settings';
+import { ValueCardPropTypes, CardPropTypes } from '../../constants/PropTypes';
+import {
+  CARD_SIZES,
+  CARD_CONTENT_PADDING,
+  VALUE_CARD_DATA_STATE,
+} from '../../constants/LayoutConstants';
+
+const { iotPrefix, prefix: carbonPrefix } = settings;
+const dsPrefix = `${iotPrefix}--data-state`;
+
+// Testing the tooltip content outside carbon proved difficult.
+// So this is exposed for testing purpose.
+export const TooltipContent = ({ tooltipContent }) => {
+  const { label, description, extraTooltipText, learnMoreURL, learnMoreText } = tooltipContent;
```

Comment:
learnMoreURL doesn't seem to have much value in a SPA.  This seems fine if you are building a website, but, I'd much rather see an onClick handler for the button/link so that an app can register what they need to do when the user does a "learn more" action.   ie, show a dialog with more information?  Jump to a page in the app?

Reply:
Yes, that makes sense. I will change it.  I chose this implementation since that is how the conceptually related learnMoreText from the RowActionError was implemented. Perhaps we should add an onClick to that one as well, so that we stay consistent?

Reply:
Shouldn't this just be defined as content by the user. So they can pass in whatever they want? @stuckless Would that be preferable?

Reply:
Now it is possible to pass a `readMoreElement` which can be a link or button with onClick or whatever is needed.

Reply:
@davidicus Yeah, I think that using a node/element here is better, since we can wire that up however we need/want.